benchmarks/dpe_bench.cc

```diff
@@ -0,0 +1,189 @@
+#include <iostream>
+#include <random>
+#include <map>
+#include <chrono>
+
+#include "hermes.h"
+#include "utils.h"
+#include "data_placement_engine.h"
+
+/* example usage: ./bin/dpe_bench -m -s 4096 */
+
+using namespace hermes;  // NOLINT(*)
+using std::chrono::time_point;
+const auto now = std::chrono::high_resolution_clock::now;
+
+enum class PlacementPolicy {
+  kRandom,
+  kRoundRobin,
+  kMinimizeIoTime,
+};
+
+#define TOTAL_TARGETS 10
+#define BLOB_RANGE testing::BlobSizeRange::kHuge
+
+#define TOTAL_NUM_BLOBS 10
+#define TOTAL_BLOB_SIZE GIGABYTES(10)
```

Comment:
The [Google style](https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros) suggests preferring `const` variables over macros.